lib/fastifySession.js

```diff
@@ -142,9 +146,14 @@ function onSend (options) {
       return
     }
 
+    let encryptedSessionId = session.encryptedSessionId;
+    if (encryptedSessionId) {
+      encryptedSessionId = `${options.cookiePrefix}${encryptedSessionId}`
+    }
```

Comment:
Sometmes the code purposefully sets `encryptedSessionId` to undefined, so this checks against that.